Uppsala/UppEt7.xml

```diff
@@ -0,0 +1,336 @@
+                                    ሥርዓተ<hi rend="partialRubric">፨</hi> በእንተ፡ ምንትኑ፡ እምቅድመ፡ ኵሉ፡ መሐሩነ፡ 
+                                    ቅዱሳን፡ አበው፡ እፎ፡ ወፂእ፡ እምዓለም፡
+                                </incipit>
+                                <explicit xml:lang="gez">
+                                    ይበርህ፡ ልቡናሁ፡ ለተጋ<unclear>ድሎ፡</unclear> <gap reason="lost"/>ድኅን፡ 
+                                    ጸሎቱ፡ እምፍልሰት<supplied reason="lost">፡</supplied> 
+                                    <gap reason="lost"/>ርህ፡ ሎቱ፡ ብርሃን፡ አምላካ<gap reason="lost"/>
+                                </explicit>
+                            </msItem>
+                            
+                        </msItem>
+                    </msContents>
+                    
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">95</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>185</height>
+                                        <width>170</width>
+                                    </dimensions>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="22">
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc>Regular hand. <bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">106</citedRange></bibl>
+                                    dates it to the 17th–18th century, whereas 
+                                    <bibl><ptr target="bm:Uhlig1988Palaographie"/><citedRange unit="page">727–728</citedRange></bibl>
+                                suggests to narrow this down to the 17th century.</desc>
+                                <date notBefore="1600" notAfter="1699">17th century.</date>
+                            </handNote>
+                        </handDesc>
+                        
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <desc type="StampExlibris">Paper sticker on the outside of the front board. It is not mentioned by 
+                                        <bibl><ptr target="bm:Zettersteen1899Upsala"/></bibl> or
+                                        <bibl><ptr target="bm:Loefgren1974Katalog"/></bibl>.</desc>
```

Comment:
Here and below: "It is not mentioned..." - this info is not necessary (most of the catalogues do not mention internal marking of the libraries, what kind of stickers existed in Zettersteen's time?). The remarks of this type make the description heavier and more difficult to look though.

Reply:
Isn't the question "what kind of stickers existed in Zettersteen's time?" exactly a reason why it may be good to encode this kind of things?

To make the description lighter, would it be possible to include a general note somewhere stating that items "xml:id so-and-so" are not mentioned in this-and-that source?

Reply:
In fact, you cannot know if the sticker was not there or Zettersteen's catalogue just does not record this detail (rather the second). If you inlcude a kind of this rule, the descriptions will be full of "negative statements", that this catalogue does not provide this and that. I have three catalogues for Abbadie collection, no one gives exact folia numbers (just to start this; of course no info on signatures/stamps). How the manuscript looked in the time of the older catalogue is the issue which we do not really approach, if we have very good digital duplicates accessible.